omnibus/config/patches/chef-dk/cli.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/lib/chef-dk/cli.rb b/lib/chef-dk/cli.rb
+index d789b075..adfc498e 100644
+--- a/lib/chef-dk/cli.rb
++++ b/lib/chef-dk/cli.rb
+@@ -95,9 +95,10 @@ BANNER
+     end
+ 
+     def show_version
+-      msg("Chef Development Kit Version: #{ChefDK::VERSION}")
++      msg("Chef Workstation Version: CHEF_WS_VERSION")
+ 
+-      ["chef-client", "delivery", "berks", "kitchen", "inspec"].each do |component|
++      msg("chef-dk version: #{ChefDK::VERSION}")
++      ["chef-apply", "chef-client", "delivery", "berks", "kitchen", "inspec"].each do |component|
```

Comment:
In the screen shot, chef-apply seems to be the chef-client version, and not the chef-runply version. Namespace conflict w/ chef-apply in the client?

Reply:
yup this is pulling the version from clients chef-apply... let me fix this to call chef-run.